spec/requests/doj_dashboard_spec.rb

```diff
@@ -1,25 +1,24 @@
 require 'rails_helper'
 
 describe '[DOJ Dashboard]', type: :request do
-  it 'prevents non-doj users from seeing the doj dashboard' do
+  it 'prevents non-DOJ users from seeing the DOJ dashboard' do
     login
     expect { visit doj_path }.to raise_error(ActionController::BadRequest)
   end
 
   describe '[with some mock data]' do
-    before :each do
-      stub_const('Constants::DEPARTMENT_BY_ORI', 'ALPHA_ORI' => 'ALPHA_DEPT', 'BRAVO_ORI' => 'BRAVO_DEPT')
-      @elvis = create(:dummy_user, first_name: "Elvis", last_name: "Presley",
-                                   email: "elvis@example.com", role: Rails.configuration.x.roles.doj,
-                                   user_id: 'elvis', ori: 'ALPHA_ORI')
+    let(:elvis) do
+      build(:dummy_user, first_name: "Elvis", last_name: "Presley",
```

Comment:
Since users have to be created before you can log in as them, there isn't much advantage to using `build` here. I'd actually argue that it would be clearer to use a `let!` block that calls `create`